TestingStaff/tstStaff.cs

```diff
@@ -240,16 +244,18 @@ public void TestFirstNameFound()
             Boolean OK = true;
 
             // Creationg test data to use with the method.
-            int StaffID = 11;
+            Int32 StaffID = 12;
 
             // Invoking the method.
             Found = StaffMember.Find(StaffID);
 
             // Checks the property.
-            if (StaffMember.FirstName != "first name") ;
+            if (StaffMember.FirstName != "Test")
```

Comment:
Strings compared like raw data types

Reply:
Followed the lab sheet for now, step by step to make sure tests pass. Will change shortly.